conf/messages

```diff
@@ -76,7 +76,7 @@ user.configuration.dataset.updated=Dataset configuration updated
 user.experience.set=Set experience of {0}
 user.experience.removed=Removed experience from {0}
 user.email.invalid=At least one Email is invalid
-user.notAuthorised=Missing Authorisation
+user.notAuthorised=You are not authorized to view this information
```

Comment:
Maybe this message could be a bit more helpful: `You are currently not authorized. Please login.`